en/docs/observe-and-manage/setting-up-cloud-native-observability-in-kubernetes.md

```diff
@@ -194,14 +194,16 @@ flow.statistics.capture_all= true
 stat.tracer.collect_payloads= true
 stat.tracer.collect_mediation_properties= true
 
-[opentracing]
+[opentelemetry]
 enable = true
-jaeger.sampler.manager_host="hostname"
-jaeger.sender.agent_host="hostname"
-```
+logs = true
+type = "jaeger"
+host = "<hostname-of-jaeger-endpoint>"
+port =  "<port-of-jaeger-endpoint>"
 
-!!! tip
-    Enter the host name of your Jaeger service as the value for `manager_host` and `agent_host` parameters. 
+# instead of ‘host’ and ‘port’, ‘url’ can be used directly in the following way.
+url =  "<url-of-jaeger-endpoint>"
```

Comment:
Should url be commented?

Reply:
Added a separate config for the `url` for better clarity via  [4b13189](https://github.com/wso2/docs-mi/pull/886/commits/4b1318965b288f195465ed09b88b254ec3a0b2eb)